packages/components/tests/integration/components/hds/table/index-test.js

```diff
@@ -243,6 +261,66 @@ module('Integration | Component | hds/table/index', function (hooks) {
       .hasText('Sorted by artist descending');
   });
 
+  test('it renders a custom sortedMessageText if supplied', async function (assert) {
+    setSortableTableData(this);
+    this.set('sortedMessageText', 'Melanie will sort it');
+
+    await render(hbsSortableTable);
+    await click('#data-test-table .hds-table__th-sort:nth-of-type(1) button');
+    assert.dom('#data-test-table caption').hasText('Melanie will sort it');
+
+    await click('#data-test-table .hds-table__th-sort:nth-of-type(1) button');
+    assert.dom('#data-test-table caption').hasText('Melanie will sort it');
+  });
+
+  test('it renders both a custom caption and a custom sortedMessageText if supplied', async function (assert) {
+    setSortableTableData(this);
+    this.set('caption', 'A custom caption.');
+    this.set('sortedMessageText', 'Melanie will sort it!');
+
+    await render(hbsSortableTable);
+    await click('#data-test-table .hds-table__th-sort:nth-of-type(1) button');
+    assert
+      .dom('#data-test-table caption')
+      .hasText('A custom caption. Melanie will sort it!');
+
+    await click('#data-test-table .hds-table__th-sort:nth-of-type(1) button');
+    assert
+      .dom('#data-test-table caption')
+      .hasText('A custom caption. Melanie will sort it!');
+  });
+
+  test('it uses a custom sort function if one is supplied', async function (assert) {
+    const mySortingFunction = (a, b) => {
+      if (a.album < b.album) {
```

Comment:
@MelSumner this is the standard sorting, so it will not detect if it works or not (in fact it was not used by the test, but the tests were passing anyway). I would suggest to use a different (custom) method, like length of the string, or last three characters)

Reply:
I had a feeling this would be the case. Will update.